docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -0,0 +1,106 @@
+# Deployment Instructions
+
+1. Clone the repo
+
+   ```bash
+   git clone https://github.com/nick-stroud/hpc-toolkit.git
+   cd hpc-toolkit
+   git checkout hcls_mega_blueprint
```

Comment:
You probably want to update the lines above, since this is going to be part of the hpc toolkit repo.

Reply:
Done.